tools/scraper/manual-data/attributes/versions.json

```diff
@@ -0,0 +1,8 @@
+{
+  "6.0 and below, 5.1 and below, 6.0 and below, 6.0": "([1-5]\\.[0-9]\\.[0-9])|(6\\.0\\.[0-9])",
+  "6.0 and below": "([1-5]\\.[0-9]\\.[0-9])|(6\\.0\\.[0-9])",
+  "5.1 and below": "([1-4]\\.[0-9]\\.[0-9])|(5\\.[0-1]\\.[0-9])",
+  "5.1 and below, 5.0 and 5.1": "([1-4]\\.[0-9]\\.[0-9])|(5\\.[0-1]\\.[0-9])",
+  " 5.1 and below": "([1-4]\\.[0-9]\\.[0-9])|(5\\.[0-1]\\.[0-9])",
```

Comment:
Leading space.

Reply:
That's there in the source data, but should hopefully be stripping this off and so working without it now